http4s/build.sbt

```diff
@@ -13,7 +13,7 @@ val `http4s-server` =
       `scala 2.12 to dotty`,
       name := "http4s-server",
       versionPolicyIntention := Compatibility.BinaryCompatible,
-      version := "6.0.0+n",
+      version := "7.0.0+n",
```

Comment:
well this didn't work, any help?

Reply:
Sorry, I know this is a bit complex... In this situation we want relax the intention to Compatibility.None (in the line above, instead of changing the version).

Reply:
Awesome, looks like it's all green now 🙌🏼